fields.yml

```diff
@@ -660,6 +660,27 @@
             City name.
           example: Montreal
     
+    - name: group
+      title: Group
+      group: 2
+      description: >
+        The group fields are meant to represent groups that are relevant to the
+        event.
+      type: group
+      fields:
+    
+        - name: id
+          level: core
```

Comment:
Perhaps approved too quick. Can we put these under `extended` for now? We can always "migrate" them to `core` later.

Reply:
Sure thing